storage/kv/merkletreekv/strkv.go

```diff
@@ -0,0 +1,35 @@
+package merkletreekv
+
+import (
+	"bytes"
+
+	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/coniks-sys/coniks-go/storage/kv"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+func StoreSTR(wb kv.Batch, str *merkletree.SignedTreeRoot) error {
```

Comment:
They would be used to load the old STRs which were deleted from the `pad.snapshot`.